### PR TITLE
AddAnimalCommand implementation added to DG

### PR DESCRIPTION
**Pull request by Blizzeracz**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -244,6 +244,40 @@ The following activity diagram summarizes what happens when a user executes a ne
...
+- `AddAnimalCommand#execute(AnimalModel model)` — Executes the command to add a specified animal to the model.
+
+Given below is an example usage scenario of the `AddAnimalCommand`:
+
+1. The user types in the `add` command with the proper cli syntax, giving a compulsory input field for each attribute. Note that the position of the cli syntax can be in any order.
+2. The command verifies the validity of the index. If the index is missing any of the required syntax, it throws a `CommandException`. Otherwise, it adds the animal with its inputted attributes to the model and returns a successful command result.
+
+The following sequence diagram shows how the `AddAnimalCommand` works:
+
+<puml src="diagrams/DeleteSequenceDiagram.puml" alt="AddSequenceDiagram" />
```
Comment (wjayee) [inline docs/DeveloperGuide.md:265]: Maybe you forgot to switch DeleteSequenceDiagram to Add !

**Review by wjayee — CHANGES_REQUESTED** [review]
LGTM! Just a small mistake in renaming

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (Blizzeracz) [reply docs/DeveloperGuide.md]: You are right! Fixed!

**Review by wjayee — APPROVED** [review]
LGTM!

**Pull request merged**